GVFS/GVFS.UnitTests.Windows/Windows/Virtualization/WindowsFileSystemVirtualizerTests.cs

```diff
@@ -535,7 +544,13 @@ public void MoveFileIntoDotGitDirectory()
 
                     // We don't expect something to rename something from outside the .gitdir to the .git\index, but this
                     // verifies that we behave as expected in case that happens
-                    mockVirtualization.OnNotifyFileRenamed("test.txt", Path.Combine(".git", "index"), isDirectory: false, notificationMask: ref notificationType);
+                    mockVirtualization.OnNotifyFileRenamed(
+                        "test.txt",
+                        Path.Combine(".git", "index"),
+                        isDirectory: false,
+                        triggeringProcessId: TriggeringProcessId,
+                        triggeringProcessImageFileName: TriggeringProcessImageFileName,
+                        notificationMask: out notificationType);

```

Comment:
Validate `notificationType` here as well.

I see there are a few more spots below.  Now that `notificationType` is an `out` parameter we should validate it everywhere we expect it to be set.

Reply:
Done locally!  Note, this was existing, but since it's a small change I incorporated it.